controllers/pokemon_controller.go

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import (
+	"jmc/bootcamp/services"
+	"jmc/bootcamp/services/io_services"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+ var _ioService services.IIoService = io_services.CsvService{PathFile: "assets/pokemon-data.csv"}
+
+func GetPokemons(c *gin.Context) {
+	pokemonList, err := _ioService.ReadFromService()
+
+	if err != nil {
+		c.IndentedJSON(http.StatusInternalServerError, gin.H{"message": err})
```

Comment:
You can also log the errors to have the message available for debugging purposes
```suggestion
		c.IndentedJSON(http.StatusInternalServerError, gin.H{"message": err})
		log.Println(err)
```